packages/cli/src/models/dependency/Dependency.ts

```diff
@@ -43,6 +43,20 @@ export default class Dependency {
     }
   }
 
+  public static hasDependenciesForDeploy(network: string): boolean {
```

Comment:
How about changing to a `getDependenciesPendingDeployment`, so we can actually show the user a list of the deps that would get deployed? Should be just a matter of changing the `find` to a `filter`.

Reply:
Its a good idea :slightly_smiling_face: